frontend/src/main/pages/MyReviews/MyReviewsIndexPage.js

```diff
@@ -1,11 +1,45 @@
+import React from "react";
+import { useBackend } from "main/utils/useBackend";
+
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import ReviewTable from "main/components/Review/ReviewTable";
+import { Button } from "react-bootstrap";
+import { useCurrentUser, hasRole } from "main/utils/currentUser";
 
 export default function MyReviewsIndexPage() {
-  // Stryker disable all : placeholder for future implementation
+  const currentUser = useCurrentUser();
+
+  const createButton = () => {
+    if (hasRole(currentUser, "ROLE_ADMIN")) {
+      return (
+        <Button
+          variant="primary"
+          href="/reviews/create"
+          style={{ float: "right" }}
+        >
+          Create Reviews
+        </Button>
+      );
+    }
+  };
+
+  const {
+    data: reviews,
+    error: _error,
+    status: _status,
+  } = useBackend(
+    // Stryker disable next-line all : don't test internal caching of React Query
+    ["/api/reviews/all"],
+    { method: "GET", url: "/api/reviews/all" },
+    [],
+  );
+
```

Comment:
It appears that this is the endpoint that admins use to get everyone's reviews.

But this is supposed to be a page that only shows `My Reviews`

So, I think that's a different endpoint.  Take a look:

<img width="576" alt="image" src="https://github.com/user-attachments/assets/14438a01-b8a6-4a16-b780-9f4e8b561991">